docs/getting-started-guides/locally.md

```diff
@@ -129,6 +132,19 @@ cluster/kubectl.sh create -f docs/user-guide/pod.yaml
 
 Congratulations!
 
+### Hacking hack-local-up cluster to test more sophisticated deployments, such as HA.
+
+To test highly available apiservers, you can modify local-up-cluster.sh as follows.
+
+1) Write a kubernetes token file which has a kubelet user with a token defined.  Lets call it /tmp/mytokens for concreteness.
+
+2) Launch the apiserver with the `--token-auth-file=/tmp/mytokens` and `--apiserver-node-port=30279` options.  Of course 30279 is just an example port.
```

Comment:
(1) lavalamp's question: NodePort is necessary because we want to test that the kubelet is properly connecting through the exposed address, 
(2) thockin's question: in this example, its token validation that is used for connecting to the service endpoint.

I guess your point around nodeport~kubernetes.local is that, w/ cluster DNS, nodeport isn't required at all?